### PR TITLE
Proposal for blockchain-carbon-accounting - Signed-off-by: Si Chen (sichen@opensourcestrategies.com)

### PR DESCRIPTION
**Pull request by opentaps**
## Short Description
CO2 emissions accounting calculations using Hyperledger Fabric

Signed-off-by: Si Chen (sichen@opensourcestrategies.com)

### CODE REVIEW

**Comment by vipinsun** [comment]
Several problems with this PR.
1. The name of the file has to be the name of your lab in the appropriate folder-
2. Has to have DCO signoff
Once these are fixed, Artem has to acknowledge that he is sponsoring lab with a comment.

**Comment by opentaps** [comment]
Sorry, Vipin, but I don't understand what you mean:


   1. The name of the file has to be the name of your lab in the
   appropriate folder-

What is the name of the file you're talking about?

Has to have DCO signoff

Who is DCO?

Thanks :)

-----
Si Chen
Open Source Strategies, Inc.

Save the Date!  2020-11-05 6 PM EST/3AM PST
Climate Accounting and Action with Hyperledger
<https://www.meetup.com/Hyperledger-Toronto/events/274129711/>

**Comment by C0rWin** [comment]
```
git rebase HEAD~6 --signoff
git push --force-with-lease origin master
```

you need to rebase and signoff all commits, above the easiest way to handle it.

File: proposal-template.md [hunk 1]
```diff
@@ -4,24 +4,29 @@ title: Lab Proposal Document
...
-- ...
+- https://github.com/opentaps
+- https://github.com/jwickers
+- https://github.com/rybalko
+- https://github.com/udosson
 
 # Sponsor
 _Provide the name of your sponsor. A sponsor is required and must be a maintainer of one of the Hyperledger projects, a TSC member, or a WG chair._
-- https://github.com/<user_id> or Name (email@example.org) - Role (e.g., "Chair of the XXX workgroup")
+- Artem Barger (BARTEM@il.ibm.com) - TSC Member
```
Comment (C0rWin) [inline proposal-template.md:29]:
```suggestion
- Artem Barger (BARTEM@il.ibm.com) - Hyperledger Fabric maintainer.
```

**Pull request closed without merging**